Bottom: 013de92444b555fb48f3ce5893b5ba7a95d668c8
Top:    c13adedf056e35f6475faa2e78ca598d935b8fc3
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-05 13:16:46 +0100

Refresh of hid-gtk-turn-on-defines-to-deb

---

diff --git a/configure.ac b/configure.ac
index 141b5d2..d2e2eeb 100644
--- a/configure.ac
+++ b/configure.ac
@@ -771,6 +771,7 @@ $GTKGLEXT_PKG_ERRORS])]
 	fi
 
 	GTK_CFLAGS="$GTK_CFLAGS -DGSEAL_ENABLE"
+#	GTK_CFLAGS="$GTK_CFLAGS -DGDK_DISABLE_DEPRECATED -DGTK_DISABLE_DEPRECATED -DGSEAL_ENABLE"
 
 	;;
